segmentstore/storage/impl/src/test/java/io/pravega/segmentstore/storage/impl/filesystem/FileSystemStorageTest.java

```diff
@@ -40,8 +55,129 @@ public void setUp() throws Exception {
 
     @After
     public void tearDown() {
-            FileHelpers.deleteFileOrDirectory(baseDir);
-            baseDir = null;
+        FileHelpers.deleteFileOrDirectory(baseDir);
+        baseDir = null;
+    }
+
+    //region Write tests with metrics checks
+    /**
+     * Tests the write() method.
+     *
+     * @throws Exception if an unexpected error occurred.
+     */
+    @Override
+    @Test(timeout = 30000)
+    public void testWrite() throws Exception {
```

Comment:
Could break the assert lines into multiple lines? They are fairly long lines now.